modules/administration-guide/examples/snip_che-getting-the-list-of-relevant-images.adoc

```diff
@@ -2,8 +2,9 @@
 ====
 [subs="+attributes,+quotes,+macros"]
 ----
-$ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/{prod-ver-patch}/eclipse-che-preview-kubernetes.v{prod-ver-patch}.clusterserviceversion.yaml | \
-  yq -r '.spec.relatedImages[]'
+$ curl -sSLo- https://raw.githubusercontent.com/eclipse-che/che-operator/{prod-ver}.x/deploy/olm-catalog/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml | \
```

Comment:
@dkwon17 wondering how this part of URL is going to be  resolved `{prod-ver}.x` e.g. 7.33.x, right?